crates/frontend/src/workspace/activity_panel/mod.rs

```diff
@@ -70,13 +71,13 @@ impl Activity {
             .event(clone!(panel, this => move |_: events::PointerDown| {                
                 panel.active_activity.set(Some(this.clone()))
             }))
-            .class_signal("has-background-white",signal::or(is_active, mouse_over.signal()))
+            
```

Comment:
Avoid noise in commit messages. Only change what needs to be changed. If there are code clean ups to be done, create another PR for that.